src/core/types/Token.ts

```diff
@@ -20,6 +20,7 @@ export interface TokenView {
   description: string;
   website: string;
   isZapable: boolean;
+  sourceIsZapper: boolean;
```

Comment:
Sounds a bit too specific, could we just have 
```suggestion
  source: SourceType;
```

Reply:
Did not wanted to propagate the typing across the app, since `SourceType` comes from SDK, and thought it was best to convert it to a primitive for the specific use case we needed, since no other source is needed till now. Maybe its worth to change it if we do have a use case in the future for the source... wdyt?

Reply:
Yeah, that's what I was thinking, in the future if more sources come we'd have to add props `sourceIsX`, `sourceIsY`, etc. If we don't foresee that happening then we can go with this, but we're reducing extensibility.

Reply:
Yeah, agree. If more sources are needed later, then it would be better to refactor this part later as you pointed rather than adding 'sourceIsX'.